app/models/occupation_standard.rb

```diff
@@ -49,6 +51,36 @@ def clone_as_unregistered!(creator_id:, organization_id:)
     end
   end
 
+  def should_generate_attachment?(kind)
+    # Occupation standard updated_at timestamp gets set milliseconds after
+    # attachment created_at timestamp.
+    !send(kind).attached? || send(kind).created_at < updated_at - 1.second
+  end
+
+  def to_csv
+    CSV.generate do |csv|
+      csv << %w(rapids_code organization_title occupation_standard_title type work_process_title work_process_description work_process_hours work_process_sort skill skill_sort)
+      occupation_standard_work_processes.includes(:work_process, :occupation_standard_skills, :skills).each do |oswp|
+        array = [rapids_code, organization_title, title, type.gsub('Standard', ''), oswp.work_process_title, oswp.work_process_description, oswp.hours, oswp.sort_order]
```

Comment:
Can we isolate this into its own method, too? Potentially handling the check for `oswp.skills` in the same place?

```
def csv_row(owsp)
 [rapids_code, organization_title, title, type.gsub('Standard', ''), oswp.work_process_title, oswp.work_process_description, oswp.hours, oswp.sort_order]
end
```

Main goal with these comments is to make this `#to_csv` easier to read. Viewing on github compounds readability issues by squeezing into more rows in the diff, but this also sets us up to more easily support a `to_xls` option later.